docs/feature_unicode.md

```diff
@@ -90,10 +90,10 @@ Unicode input in QMK works by inputting a sequence of characters to the OS, sort
 
 The following input modes are available:
 
-* **`UC_OSX`**: macOS built-in Unicode hex input. Supports code points up to `0xFFFF` (`0x10FFFF` with Unicode Map).
+* **`UC_MAC`**: macOS built-in Unicode hex input. Supports code points up to `0xFFFF` (`0x10FFFF` with Unicode Map).
```

Comment:
I modified this page quite a bit in #6286. I don't know much about this whole merging, rebasing, whatever process but I hope these conflicts don't cause too much of a headache.

Reply:
Oh don't worry, that's already been taken care of. There was only one conflict — on this exact line.

Reply:
Since your PR was merged into `master`, and `future_unformatted` was rebased onto `master` recently, these commits come after your commit, so everything's cool 😃

Reply:
I really should find some time to learn how this more advanced git stuff works... it's Dark Magic to me 😆